src/HashingLogic.ts

```diff
@@ -67,11 +67,59 @@ export const hashAttestation = (attestation: IAttestationData) => {
  * as the leaves being sorted by type and mapped into hash Buffers.
  */
 export const getMerkleTree = (attestations: IAttestationData[]) => {
-  const leaves = sortBy(attestations, ['type'])
-    .map(hashAttestation)
-    .map(hexStr => Buffer.from(hexStr, 'hex'))
-  return new MerkleTree(leaves, x => Buffer.from(hashAttestation(x), 'hex'))
+  const leaves = attestations
+  .map(hashAttestation)
+  .sort()
+  .map(hexStr => Buffer.from(hexStr, 'hex'))
+  // const leaves = sortBy(attestations, ['type'])
+  //   .map(hashAttestation)
+  //   .map(hexStr => Buffer.from(hexStr, 'hex'))
+  return new MerkleTree(leaves, x => Buffer.from(keccak256(x), 'hex'))
 }
+/**
+ * verify
+ * @desc Returns true if the proof path (array of hashes) can connect the target node
+ * to the Merkle root.
+ * @param {Object[]} proof - Array of proof objects that should connect
+ * target node to Merkle root.
+ * @param {Buffer} targetNode - Target node Buffer
+ * @param {Buffer} root - Merkle root Buffer
+ * @return {Boolean}
+ * @example
+ * const root = tree.getRoot()
+ * const proof = tree.getProof(leaves[2])
+ * const verified = tree.verify(proof, leaves[2], root)
+ * 
+ * standalone verify function taken from https://github.com/miguelmota/merkletreejs
+ */
+export const verify = (
```

Comment:
I think this needs to be named something more descriptive since it's no longer tied to a `MerkleTree` instance.

Reply:
👍